src/pybind/mgr/dashboard/frontend/package.json

```
@@ -5,7 +5,7 @@
   "scripts": {
     "ng": "ng",
     "start": "ng serve --host 0.0.0.0",
-    "build": "ng build",
+    "build": "cp src/environments/environment.tpl.ts src/environments/environment.prod.ts && cp src/environments/environment.tpl.ts src/environments/environment.ts && ng build && node ./environment.build.js",
```

What about the following:
```suggestion
    "start": "npm run env_build && ng serve --host 0.0.0.0",	
    "build": "npm run env_build && ng build",
    "env_build": "cp src/environments/environment.tpl.ts src/environments/environment.prod.ts && cp src/environments/environment.tpl.ts src/environments/environment.ts  && node ./environment.build.js"
```
with this we would get both `start` and `build` working with the new environment.build